src/security.cs

```diff
@@ -891,4 +891,36 @@ interface SecKeyKeyExchangeParameter {
 
 		NSData SharedInfo { get; set; }
 	}
+
+#if IOS
+	[iOS (8,0)][NoTV][NoWatch][NoMac]
+	[Internal][Static]
+	interface SecSharedCredentialKeys {
+		[Field ("kSecAttrServer")]
+		NSString ServerKey { get; }
+
+		[Field ("kSecAttrAccount")]
+		NSString AccountKey { get; }
+
+		[Field ("kSecSharedPassword")]
+		NSString PasswordKey { get; }
+
+		[Field ("kSecAttrPort")]
+		NSString _PortKey { get; }
+	}
+
+	[iOS (8,0)][NoTV][NoWatch][NoMac]
+	[StrongDictionary ("SecSharedCredentialKeys")]
+	interface SecSharedCredentialInfo {
+
+		string Server { get; set; }
+
+		string Account { get; set; }
+
+		string Password { get; set; }
+
+		[Internal]
+		NSNumber _Port { get; set; }
```

Comment:
Any reason for nor just using

```
int Port { get; set; }
```

here?

Reply:
it's optional and I thought `int?` did not work or was that added recently?

Reply:
if you use any of the primitives it will be converted to its nullable version IIRC

Reply:
so doing `int` should output `int?` and if you set it to `null` it will remove the key from the dictionary